psalm.xml

```diff
@@ -28,4 +28,12 @@
         <function name="var_dump"/>
         <function name="print_r"/>
     </forbiddenFunctions>
+    <issueHandlers>
+        <TooManyTemplateParams>
+            <errorLevel type="suppress">
+                <file name="src/Factory/SynonymFactoryInterface.php"/>
```

Comment:
I have done this before. Actually I just tend to suppress both `Factory` and `Repository` directories for this particular issue